### PR TITLE
Phase 3 Funding: Craig | Widget-based app for optical data reduction and stellar photometry

### PR DESCRIPTION
**Pull request by mwcraig**
This very rough draft outlines an idea for a widget-based app aimed at novice users (including amateurs) for optical data reduction, stellar photometry, and some basic analysis tools.

### CODE REVIEW

**Comment by kelle** [comment]
I am interested in being involved in this effort. I would like to provide feedback and figure out how the final product can integrate with the rest of the Learn ecosystem.  My contribution would be covered by my own funding request (#285).

**Comment by pllim** [comment]
I would like @eteq to chime in here because STScI has Jdaviz as well.

**Comment by mwcraig** [comment]
> I would like @eteq to chime in here because STScI has Jdaviz as well.

It would be really nice to start with something like that. No need to reinvent the wheel.

**Comment by jdswinbank** [comment]
Because the estimated total of all the proposals is a bit higher than the amount available for allocation, we suggest that you (@mwcraig ) provide a budget range - i.e., a "minimum for this project to be viable" and "maximum you could use effectively for this project".

File: finance/proposal-calls/cycle3/optical-reduction-photometry-entry-level-gui-app.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+### Title
+
+Develop a Jupyter widget-based app for reducing and performing stellar photomtery on optical images
```
Comment (hamogu) [inline finance/proposal-calls/cycle3/optical-reduction-photometry-entry-level-gui-app.md:3]:
```suggestion
Develop a Jupyter widget-based app for reducing and performing stellar photometry on optical images
```

File: finance/proposal-calls/cycle3/optical-reduction-photometry-entry-level-gui-app.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+While astropy already provides a mechanism for scripting those activities, it is painful to have to write code every time. It can be done in a way that novices can use it (see the [photometry notebooks here](https://github.com/mwcraig/ast266-notes/tree/main/notebook-templates) for an example), but is cumbersome.
+
+Doing something simple like generating a time series requires substantial boilerplate code. While tedious for someone experienced, that boilerplate is intimidating to new comers.
+
+The community this potentially impacts are amateur astronomers, those teaching astronomy in high school or at the undergraduate level, and potentially those running REU programs.
+
+### Project / Work
+
+To be clean, none of the ideas here are really new. For example, [LEMON](https://github.com/vterron/lemon), presented at the first PyAstro conference, did some of these things without a GUI. It has not had a release in 5 years, though.
```
Comment (hamogu) [inline finance/proposal-calls/cycle3/optical-reduction-photometry-entry-level-gui-app.md:21]:
```suggestion
To be clear, none of the ideas here are really new. For example, [LEMON](https://github.com/vterron/lemon), presented at the first PyAstro conference, did some of these things without a GUI. It has not had a release in 5 years, though.
```

**Comment by aaryapatil** [comment]
Please react to this comment to vote on this proposal (👍, 👎, or no reaction for +0)

**Comment by kelle** [comment]
Congratulations!  The community has expressed support for this FR, and the Finance Committee has determined it can be funded.  This FR has been approved for $32,000, to be funded from the Moore grant.

@mwcraig is the Finance Committee member who will be supporting this project. They will update this PR with the amount and then merge it, as well as being in contact with you to set up any necessary agreements and relevant Open Collective financial arrangements. They will also open a new issue where progress on this project can be discussed.

**Pull request merged**